### DISCUSSION TITLE
Thermodynamic integration for a certain atom in a system

### DISCUSSION THREAD

**Discussion by wxwth** [deepmodeling/deepmd-kit#3006] [Q&A]
Dear DeePMD developers and users,
  I am trying to perform (Hamiltonian) thermodynamic integration for a certain atom in a system, e.g., for Na+ in a system containing one Na+ and several bulk water molecules, by using the deep potential. Could the LAMMPS installed in DeePMD-kit perform such a calculation? If yes, are there any examples, or which keywords in LAMMPS I should use? Thanks in advance!

**Comment by wanghan-iapcm** [comment]
you may want to take a look at the lammps input scripts generated by the [dpti](https://github.com/deepmodeling/dpti)
